plugins/inputs/opentelemetry/grpc_services.go

```diff
@@ -7,14 +7,15 @@ import (
 	"github.com/influxdata/influxdb-observability/common"
 	"github.com/influxdata/influxdb-observability/otel2influx"
 	"go.opentelemetry.io/collector/model/otlpgrpc"
-	"go.opentelemetry.io/collector/model/pdata"
 )
 
 type traceService struct {
 	converter *otel2influx.OtelTracesToLineProtocol
 	writer    *writeToAccumulator
 }
 
+var _ otlpgrpc.TracesServer = (*traceService)(nil)
```

Comment:
This makes it easier for someone (likely myself) to easily detect changes to the interface, which cause the implementation to be adjusted. It was a task in this PR, so I left the helpers in there.

It isn't an uncommon pattern, but I understand that it may look ugly to you.

Reply:
Didn't know... It's fine with me then... :-)